app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -303,5 +280,5 @@ void medRegistrationWorkspace::updateFromRegistrationSuccess(medAbstractData *ou
 
 void medRegistrationWorkspace::removeSelectorInternToolBox()
```

Comment:
selector is already destroyed by `medSelectorWorkspace`. I don't think we need this function anymore.